packages/stdlib/src/ring1/async.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
```

Comment:
Given that `async` is almost a keyword in JS, it seems like asking for trouble to name a module simply `async`. But I cannot think of any actual problem it causes.

Reply:
Let's call it `async-tools.js`!